lib/steps.js

```diff
@@ -112,7 +113,8 @@ Given(/I start the interview at "([^"]+)"(?: in lang "([^"]+)")?/, {timeout: -1}
   while ( num_attempts < max_load_attempts ) {
 
     num_attempts++;
-    let timeout_message = `Failed to load "${ file_name }" after ${ num_attempts } tries. `
+    let url = `${ await get_base_interview_url() }${ file_name }`;
+    let timeout_message = `Failed to load "${ file_name }" at URL ${ url } after ${ num_attempts } tries. `
```

Comment:
Just fyi, this is technically a GitHub secret as well. Do we want to save it to files? I can't see why someone would want to keep their server url secret, but it's inconsistent treatment of secrets. Should we use the name of the secret instead? Wish GitHub had non-secret config env vars.

Reply:
Ugh, didn't think about that. I made this change just because it's hard to get the URL right if you can't see it, especially if you're running on localhost (you need the `http://`, which isn't clear). It's probably okay to revert, this seemed hacky anyway.